src/api/app/controllers/webui/users_controller.rb

```diff
@@ -136,6 +147,32 @@ def change_password
 
   private
 
+  def adjust_filters
```

Comment:
We're gonna try to move this to a model, so its easier to test

Reply:
service object or presenter. IMO, This information has nothing to do with a model.